proposals/lip-0057.md

```diff
@@ -72,6 +72,12 @@ By setting the BFT weights proportional to validator weight instead of uniform B
 
 In the current system of uniform BFT weights this is not the case. The set of 34 lowest-weight validators may have significantly less than 1/3 of the total validator weight although their total BFT weight is more than 1/3. This means that quite a lot less than 1/3 of the total staked amount for active validators is sufficient to control more than 1/3 of BFT weight, which would be sufficient to perform an attack on the network. Choosing BFT weights proportional to validator weight avoids this issue.
 
+### Bootstrap and Hybrid Period
+
+The bootstrap period, defined in [LIP 0034][lip-0034#bootstrap-period], is the initial period when the PoS validator selection mechanism is not active and blocks are generated by a fixed set of validators (specified in genesis block), called *initial validators*. This period should be long enough to prevent long range attacks, as explained in [LIP 0034][lip-0034#bootstrap-period]. In LIP 0034, we assumed that after the bootstrap period ends, then all validators are selected according to the PoS selection mechanism. One drawback of this approach is that after the end of the bootstrap period, the change of validators might be too abrupt, possibly leading to breaking the BFT security assumptions and finalizing contradicting blocks or inability of the chain to produce certificates. To avoid those issues, in LIP 34 we just did not allow blocks to be finalized (and therefore produce certificates) during the bootstrap period. This approach has some drawbacks, for example, it does not allow sidechains to be interoperable during their bootstrap period, which is not desirable.
```

Comment:
It would be easier to make suggestions if you have one sentence per line.

Note that long range attacks are not an issue for this migration any more as we do not reset the votes and make people re-vote.

> To avoid those issues, in LIP 34 we just did not allow blocks to be finalized (and therefore produce certificates) during the bootstrap period.

I would remove "just" and the part about certificates as those do not exist in Lisk Core v3.

Reply:
Rephrased, have a look.